e2e/samples/httpbin.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright Istio Authors
```

Comment:
I have just updated a different httpbin.yaml that actually uses 2 different ports (8000 and 80) in both service ports and workload ports, that is better than this as it uses the same workload port 80, you can remove this and just use the one I updated.